qcodes/tests/dataset/test_database_creation_and_upgrading.py

```diff
@@ -922,3 +923,31 @@ def test_getting_db_version(version):
 
     assert db_v == version
     assert new_v == LATEST_VERSION
+
+
+@pytest.mark.parametrize('db_file',
+                         ['empty',
+                          'some_runs'])
+
```

Comment:
this empty line should not be here, right?

Reply:
Fixed. Thanks